node/src/BaseClient.ts

```diff
@@ -1393,6 +1394,27 @@ export class BaseClient {
         return this.createWritePromise(createHVals(key));
     }
 
+    /**
+     * Returns the string length of the value associated with `field` in the hash stored at `key`.
+     *
+     * See https://valkey.io/commands/hget/ for details.
+     *
+     * @param key - The key of the hash.
+     * @param field - The field in the hash.
+     * @returns The string length or `0` if `field` or `key` does not exist.
+     *
+     * @example
+     * ```typescript
+     * // Example usage of the hget method on an-existing field
+     * await client.hset("my_hash", "field", "value");
```

Comment:
hset takes a Record as its second arg
```suggestion
     * await client.hset("my_hash", {"field": "value"});
```